pallets/capacity/src/lib.rs

```diff
@@ -212,6 +249,36 @@ pub mod pallet {
 	pub type UnstakeUnlocks<T: Config> =
 		StorageMap<_, Twox64Concat, T::AccountId, UnlockChunkList<T>>;
 
+	/// stores how many times an account has retargeted, and when it last retargeted.
+	#[pallet::storage]
+	#[pallet::getter(fn get_retargets_for)]
+	pub type Retargets<T: Config> = StorageMap<_, Twox64Concat, T::AccountId, RetargetInfo<T>>;
+
+	/// Information about the current reward era. Checked every block.
+	#[pallet::storage]
+	#[pallet::whitelist_storage]
+	#[pallet::getter(fn get_current_era)]
+	pub type CurrentEraInfo<T: Config> =
+		StorageValue<_, RewardEraInfo<RewardEra, BlockNumberFor<T>>, ValueQuery>;
+
+	/// Reward Pool history is divided into chunks of size RewardPoolChunkLength.
+	/// ProviderBoostHistoryLimit is the total number of items, the key is the
+	/// chunk number.
+	#[pallet::storage]
+	#[pallet::getter(fn get_reward_pool_chunk)]
+	pub type ProviderBoostRewardPools<T: Config> =
+		StorageMap<_, Twox64Concat, u32, RewardPoolHistoryChunk<T>>;
+
+	/// How much is staked this era
+	#[pallet::storage]
+	pub type CurrentEraProviderBoostTotal<T: Config> = StorageValue<_, BalanceOf<T>, ValueQuery>;
+
+	/// Individual history for each account that has Provider-Boosted.
+	#[pallet::storage]
+	#[pallet::getter(fn get_staking_history_for)]
```

Comment:
I believe the following might be irrelevant if we decide to remove the macro getter functions.

Since we have two types of staking, boost and maximum, it would be helpful to be specific about which type of staking we are referring to. Therefore, I recommend renaming `get_staking_history_for` to `get_boost_staking_history_for`.

Reply:
I will open a PR to remove these getters